endpoints/dependabot.ts

```diff
@@ -0,0 +1,36 @@
+import got from '../libs/got'
+import {
+  badgenServe,
+  BadgenServeMeta as Meta,
+  BadgenServeHandlers as Handlers,
+  BadgenServeHandlerArgs as Args
+} from '../libs/badgen-serve'
+
+export const meta: Meta = {
+  title: 'Dependabot',
+  examples: {
+    '/dependabot/dependabot/dependabot-core/?icon=dependabot': 'status',
+    '/dependabot/greenkeeperio/greenkeeper/<repo_id>?icon=dependabot': 'status (private repo)'
```

Comment:
`<repo_id>` in example might make user confused

Reply:
Fair point, addressed your concerns in the latest commit 😃